src/canvas/Canvas.ts

```diff
@@ -1055,7 +1056,7 @@ export class Canvas extends SelectableCanvas implements CanvasOptions {
       );
       if (target === this._activeObject && (corner || !grouped)) {
         this._setupCurrentTransform(e, target, alreadySelected);
-        const control = target.controls[corner],
+        const control = corner?.control,
```

Comment:
i hate the question mark because the need of transpiling it. in case can you do `corner && corner.control` ?